app/src/debug/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-
+    <meta-data android:name="io.fabric.ApiKey" android:value="fabric_api_key" />
```

Comment:
why?

Reply:
that was an attempt to resolve build issue... mentioned in readme, but now its redundant. Updated according to review and squashed into single commit.